### ISSUE TITLE
page stays the same after exporting content to USB drive

### ISSUE THREAD

**Issue by lyw07** [learningequality/kolibri#5621]
### Observed behavior

After clicking exporting content to USB drive, the page got refreshed but stayed at the export content page. 

### Expected behavior

Users should be directed to the channel list page after the content gets successfully exported.

### User-facing consequences

Users would think that the content hasn't been exported

### Errors and logs

nothing wrong from the log file.
```
INFO     Exporting channel database for channel id a9b25ac9814742c883ce1b0579448337 to /KOLIBRI_DATA
INFO     Exporting content for channel id a9b25ac9814742c883ce1b0579448337 to /KOLIBRI_DATA
```

### Steps to reproduce

1. import some content from studio
2. export the content to USB drive

### Context

kolibri version: latest pex from 0.12.y release branch
operating system: OSX
browser: chrome

**Comment by lyw07** [comment]
or is this expected?

**Comment by lyw07** [comment]
fixed in #5542
